kolibri/content/models.py

```diff
@@ -170,11 +177,14 @@ def save(self, *args, **kwargs):
             md5 = hashlib.md5()
             for chunk in self.content_copy.chunks():
                 md5.update(chunk)
-
             self.checksum = md5.hexdigest()
             self.available = True
             self.file_size = self.content_copy.size
-            self.extension = os.path.splitext(self.content_copy.name)[1]
+            # if feed a format that doesn't exist yet will break the app
+            # not sure how we want to handle this yet
```

Comment:
The client side shouldn't allow uploading of files with extensions that aren't associated with non-suppliemental presets. If we do receive one, we should return an error.